shortcodes/ucf-faq-list-shortcode.php

```diff
@@ -157,11 +157,22 @@ public static function shortcode( $atts ) {
 				'topic_class'    => 'h5',
 			), $atts, 'ucf-faq-topic-list' );
 
-			$topics = get_terms( 'topic', array(
-				'post_type' => array( 'faq' )
-			) );
+			$args = array(
```

Comment:
`post_type` and `posts_per_page` aren't valid arguments for `get_terms()`; see https://developer.wordpress.org/reference/classes/wp_term_query/__construct/

Since both of those arguments are likely being ignored by `get_terms()` right now, you could probably remove `$args` altogether here (and not pass it in to `get_terms()` on line 166)